### ISSUE TITLE
As Em, I want to know more about this website and who made it.

### ISSUE THREAD

**Issue by cpenalosa** [newjersey/dol-mcnj-main#62]
As Em,
I want to learn more about the Training Explorer and who built it,
So that I can start to build trust with the information I'm seeing on the website.

**GIVEN** I'm on the home page
**THEN** I will see a link to Contact Us in the footer
**IF** I click on the link
**THEN** I will see information about how the Training Explorer website was built and how to get in touch if I have more questions.

Inspiration: https://business.nj.gov/contact-us

[Desktop](https://www.figma.com/file/E93aghu0LH4kXQWReLfNNY/D4AD---Training-Explorer?node-id=1655%3A0)
[Tablet](https://www.figma.com/file/E93aghu0LH4kXQWReLfNNY/D4AD---Training-Explorer?node-id=4278%3A9451)
[Mobile](https://www.figma.com/file/E93aghu0LH4kXQWReLfNNY/D4AD---Training-Explorer?node-id=4278%3A9467)

**Comment by namanaman** [comment]
@cpenalosa 
- I think the headers might be a little confusing on this page given that there is not too much content to label. "Have questions about finding training?..." seems to be the heading for the content underneath it, which is "About this site" and not related. It's also a little repetitive with the "Contact the Training Explorer team with your question" header below. I was thinking to simplify and just have 2 headers, one for "About", and the other for "Contact".

Current:
<img width="1273" alt="Screen Shot 2022-03-24 at 12 07 45 PM" src="https://user-images.githubusercontent.com/9057802/159961159-31fe0625-85d9-43f0-93ec-213638559599.png">

Proposed:
<img width="1217" alt="Screen Shot 2022-03-24 at 12 10 43 PM" src="https://user-images.githubusercontent.com/9057802/159961161-d3bf4477-ca2c-4b0d-b482-62a785e5db17.png">

- Is there a specific Google account to set up the form under?

**Comment by cpenalosa** [comment]
Good catch! I think I was workshopping some different content headers and left the draft version on the figma account. Let's go with the proposed version you have.
